### PR TITLE
Gen 1: 2022 OU tier shift

### PR DESCRIPTION
**Pull request by PlagueVonKarma**
TLDR; Victreebel UU, Slowbro/Lapras UUBL
This is important for the upcoming UU Open and UU Snake Draft! 

https://www.smogon.com/forums/threads/rby-ou-viability-rankings.3685861/page-5#post-9254185
The new 2022 RBY OU VR was just released, and some tier shifts have occurred;
- Lapras and Victreebel are dropping to UU properly now. 
- Slowbro is...being Slowbro.

Slowbro is in its own rank on the VR, but it's juuust slim enough for it to drop to UU, according to phoopes and vapicuno, at least by the quantitative metrics we're using. There's a deep-seated tradition of "The Slowbro Standard" but it's finally crumbled. 

**However, the RBY UU Council has quickbanned Lapras and Slowbro** because, well, yeah...they're very meta-warping, so on, blah blah. Victreebel lives though! There'll be a post on this soon, I'm just making this now so it all goes smoothly. 

Given the OU by Technicality stuff, I've simply changed it to UUBL for Gen 1 Random Battles, it functionally doesn't change Lapras but does buff Slowbro. 

I don't know how the teambuilder processes new tiers, I remember having to flick a switch or something for (OU) and NUBL to show up in the past. I think it was on the client? Is someone able to check on that for me? 

Thank you for your time!

### CODE REVIEW

**Comment by KrisXV** [comment]
This looks fine, but since it touches a randbats file I'll leave it to @AnnikaCodes to merge this

**Review by Marty-D — APPROVED** [review]

**Comment by PlagueVonKarma** [comment]
You may want to talk with Amaranth or someone else on the RBY Council; apparently, there's some dispute over Slowbro. It's an idiotic mess, sorry to drag you into it.

File: data/mods/gen1/formats-data.ts [hunk 1]
```diff
@@ -414,7 +414,7 @@ export const FormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 	slowbro: {
 		randomBattleMoves: ["amnesia", "surf", "thunderwave"],
 		exclusiveMoves: ["blizzard", "psychic", "rest", "rest"],
-		tier: "OU",
+		tier: "UUBL",
```
Comment (AnnikaCodes) [inline data/mods/gen1/formats-data.ts:417]: If I understand it correctly, Slowbro is banned from UU but usable in OU regardless of how it's classified here, so I'm not worried about adjudicating a dispute here since it's purely cosmetic and we can easily push something to change it.

**Comment by AnnikaCodes** [comment]
Thanks!
